CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## 1.0.0-preview.1 (2021-05-28)
+
+- Initial preview release of the v6 version of the Autorest Typescript/Javascript SDK Generator.
```

Comment:
I think we should mention that this package is a rebranding of the older @microsoft one. cc @ramya-rao-a

Reply:
Added that line